site2/docs/functions-develop.md

```diff
@@ -35,6 +35,20 @@ def process(input):
 ```
 For complete code, see [here](https://github.com/apache/pulsar/blob/master/pulsar-functions/python-examples/native_exclamation_function.py).
 
+Note that functions can be written in python2 or python3, but pulsar
+currently only looks for "python" as the interpreter to execute them.
+
+A recent Ubuntu system may have only "python3" but not "python", in which
+case functions will fail to start.  As a workaround you can create a symlink, but beware this has some
+[risks](https://askubuntu.com/questions/320996/how-to-make-python-program-command-execute-python-3#answer-475815):
+
+```bash
+sudo update-alternatives --install /usr/bin/python python /usr/bin/python3 10
+```
+
+If you choose to do this, be careful not to install any other package which
+depends on "python" (2.x)
```

Comment:
> Note 
> You can write Pulsar Functions in python2 or python3. However, Pulsar only looks for `python` as the interpreter.

> If you're running Pulsar Functions on Ubuntu system that only supports `python3`, you might fail to
start the functions. In this case, you can create a symlink. However, such action
has potential [risks](https://askubuntu.com/questions/320996/how-to-make-python-program-command-execute-python-3#answer-475815).

```bash
sudo update-alternatives --install /usr/bin/python python /usr/bin/python3 10
```
> If you create a symlink, you'd better not install any other package that depends on "python" (2.x).

Reply:
A tiny suggestion on this: If we can summarize the risks briefly, we can use a brief summary, and not use external links.
Reason: The external link might change or someone might delete the Q&A in this link, then the link in our docs will be broken, it's hard to maintain external links.

Reply:
It could link to #5518 on github instead.  Actually, I'm hoping #5518 will accept a new config option for setting the path to the python interpreter, in which case we would document that instead.

Reply:
yes, sure.